libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -375,7 +421,19 @@ export class XCCDFResultsMapper extends BaseConverter {
                 )
               })
             },
-            refs: [],
+            refs: [
```

Comment:
from xccdf-scc-rhel8-hdf.json

![image](https://github.com/mitre/heimdall2/assets/2015821/b6f6d2b1-6654-4d1b-80f5-559e6f4d1829)

your sample outputs don't match the schema for some reason

https://github.com/mitre/heimdall2/blob/master/libs/inspecjs/src/generated_parsers/v_1_0/exec-json.ts#L244

should probably look something like `refs: [ref: [{publisher: blah, ...}]]`

Reply:
That data structure is a bit confusing... but I think I matched the intent now.